app/src/main/kotlin/at/bitfire/davdroid/resource/LocalAddressBook.kt

```diff
@@ -103,11 +104,11 @@ open class LocalAddressBook @AssistedInject constructor(
     val includeGroups
         get() = groupMethod == GroupMethod.GROUP_VCARDS
 
-    @Deprecated("Local collection should be identified by ID, not by URL")
-    override var collectionUrl: String
-        get() = AccountManager.get(context).getUserData(addressBookAccount, USER_DATA_URL)
-                ?: throw IllegalStateException("Address book has no URL")
-        set(url) = AccountManager.get(context).setAndVerifyUserData(addressBookAccount, USER_DATA_URL, url)
+    override var dbCollectionId: Long?
+        get() = accountManager.getUserData(addressBookAccount, USER_DATA_COLLECTION_ID)?.toLongOrNull() ?: throw IllegalStateException("Address book has no collection ID")
```

Comment:
Do we need this exception? The other `LocalCollection`s just return `null`.

Reply:
I don't see why. It's been there before, so I simply left it. I will remove it.